### DISCUSSION TITLE
useBounds() hook undefined

### DISCUSSION THREAD

**Discussion by dreammonkey** [pmndrs/drei#953] [Q&A]
I'm trying to implement the useBounds hook as shown in the example [here](https://github.com/pmndrs/drei#bounds)

But it seems the hook is undefined, am I doing something wrong ?

https://codesandbox.io/s/optimistic-brahmagupta-skj3vz?file=/src/App.js

**Comment by drcmda** [comment]
hooks can only be used within their context, useBounds needs to be within the bounds component which gives it context.
